openwisp_controller/urls.py

```diff
@@ -50,6 +50,12 @@
         'app': 'openwisp_controller.config',
         'include': {'module': '{app}.api.urls', 'namespace': 'controller_config'},
     },
+    # openwisp_controller.connection
+    {
+        'regexp': r'^',
+        'app': 'openwisp_controller.connection',
+        'include': {'module': '{app}.api.urls', 'namespace': 'controller_connection'},
```

Comment:
namespace should be `connection_api`, see https://github.com/openwisp/openwisp-controller/blob/797b41e1048f9b8c2939298c2f506a55e811e46b/openwisp_controller/urls.py#L51
https://github.com/openwisp/openwisp-controller/blob/797b41e1048f9b8c2939298c2f506a55e811e46b/openwisp_controller/urls.py#L39